inference-engine/src/vpu/graph_transformer/src/stages/strided_slice.cpp

```diff
@@ -4,15 +4,32 @@
 
 #include <vpu/frontend/frontend.hpp>
 
+#include <vpu/stages/post_op_stage.hpp>
+
 #include <memory>
 #include <vector>
 
-#include <vpu/stages/post_op_stage.hpp>
-
 namespace vpu {
 
 namespace {
 
+int maskStrToInt(std::string mask) {
```

Comment:
`mask` cannot have spaces, or they will be processed correctly?

Reply:
No, it can't. Mask has pattern `x,x,x,x`.